system/src/system_cloud_internal.cpp

```diff
@@ -475,6 +478,24 @@ void SystemEvents(const char* name, const char* data)
                 System.reset();
         }
     }
+    if (!strncmp(name, KEY_RESTORE_EVENT, strlen(KEY_RESTORE_EVENT))) {
```

Comment:
What's the usecase for this? If your key/server address information was corrupted/removed then you won't be able to connect to server at all. And if you are already connected and you are able to receive this event from the server, it means that you probably have valid public key and server address :sweat_smile: